Definition of coordinate systems : add formulas (#60)
ices-eg/wg_WGFAST: SONAR-netCDF4/docs/tableBeamGroup1.adoc

 3+|{attr}:units = "arc_degree" 
 3+|{attr}float :valid_range = 0.0, 360.0 
 

>  |{var}float rx_beam_rotation_alpha(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise rotation rotation about the _z_-axis of the platform coordinate system needed to give the rx beams coordinate system.

Reviewer: ```suggestion
 |{var}float rx_beam_rotation_alpha(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise rotation about the _z_-axis of the platform coordinate system needed to give the receive beams coordinate system.
```

 3+|{attr}:long_name = "rx beam angular rotation about the _z_ axis"
 3+|{attr}:units = "arc_degree" 
 3+|{attr}float :valid_range = −180.0, 180.0 
 

>  |{var}float rx_beam_rotation_beta(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _x_-axis of the platform coordinate system needed to give the rx beams coordinate system.

Reviewer: ```suggestion
 |{var}float rx_beam_rotation_beta(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _y_-axis of the platform coordinate system needed to give the rx beams coordinate system.
```

>  3+|{attr}:long_name = "rx beam angular rotation about the _x_ axis"

Reviewer: ```suggestion
 3+|{attr}:long_name = "rx beam angular rotation about the _y_ axis"
```

Author: updated in commit 009e629

 3+|{attr}:units = "arc_degree" 
 3+|{attr}float :valid_range = −180.0, 180.0 
 

>  |{var}float rx_beam_rotation_gamma(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _z_-axis of the platform coordinate system needed to give the rx beams coordinate system.

Reviewer: ```suggestion
 |{var}float rx_beam_rotation_gamma(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _x_-axis of the platform coordinate system needed to give the rx beams coordinate system.
```

Author: updated in commit 009e629

>  3+|{attr}:long_name = "rx beam angular rotation about the _z_ axis"

Reviewer: ```suggestion
 3+|{attr}:long_name = "rx beam angular rotation about the _x_ axis"
```

Author: updated in commit 009e629

 3+|{attr}:units = "arc_degree" 
 3+|{attr}float :valid_range = −180.0, 180.0 
 
 |{var}float tx_beam_rotation_alpha(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise rotation  rotation about the _z_-axis of the platform coordinate system needed to give the tx beams coordinate system.
 3+|{attr}:long_name = "tx beam angular rotation about the _z_ axis"
 3+|{attr}:units = "arc_degree"
 3+|{attr}float :valid_range = −180.0, 180.0

>  |{var}float tx_beam_rotation_beta(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _x_-axis of the platform coordinate system needed to give the tx beams coordinate system.

Reviewer: ```suggestion
 |{var}float tx_beam_rotation_beta(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _y_-axis of the platform coordinate system needed to give the tx beams coordinate system.
```

 3+|{attr}:long_name = "tx beam angular rotation about the _x_ axis"
 3+|{attr}:units = "arc_degree"
 3+|{attr}float :valid_range = −180.0, 180.0

>  |{var}float tx_beam_rotation_gamma(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _z_-axis of the platform coordinate system needed to give the tx beams coordinate system.

Reviewer: ```suggestion
 |{var}float tx_beam_rotation_gamma(ping_time, beam) |M |The intrinsic _z_–_y_’–_x_” clockwise about the _x_-axis of the platform coordinate system needed to give the tx beams coordinate system.
```

Author: I started to correct angles and I just realized that in case of Tait–Bryan angles is more often written as theta, phi, psi rather than alpha, beta, gamma.
Wouldn't it be better to use the same convention as wikipedia, or even call them with yaw,pitch,roll

Reviewer: I agree. I suggest theta, phi, and psi rather than yaw, pitch, and roll. This because these are not the yaw, pitch, and roll of the platform, but a rotation of the PCS to get the BCS. Does that make sense?

Author: Good for me, I will work on that

Author: Just one question : should I mark all comments as resolved and make a new commit for this ?

Reviewer: I have no preference. Which every way suits you.

>  3+|{attr}:long_name = "tx beam angular rotation about the _z_ axis"

Reviewer: ```suggestion
 3+|{attr}:long_name = "tx beam angular rotation about the _x_ axis"
```

 3+|{attr}:long_name = "roll angle" 
 3+|{attr}:coordinates = "ping_time platform_latitude platform_longitude" 
 

>  |{var}float platform_vertical_offset(ping_time) |M |Distance from the platform reference point to the water line (distance are positives downwards). For ships and similar, this is called heave added to the dynamic draught at the time of the ping but the concept applies equally well to underwater vehicle depth.

Reviewer: ```suggestion
 |{var}float platform_vertical_offset(ping_time) |M |Distance from the platform reference point to the water line (distance are positives downwards). For ships and similar, this is called heave and is added to the dynamic draught at the time of the ping but the concept applies equally well to underwater vehicle depth.
```

 3+|{attr}:units = "m" 
 3+|{attr}:coordinates = "ping_time platform_latitude platform_longitude" 
 

>  |{var}float waterline_to_chart_datum(ping_time) |O |Vertical translation vector at the time of the ping matching the distance from the water line to the chart data reference (typically Lowest Astronomical Tide or Mean Sea Level). This variable is the vector contains the tide and allow to position samples in an absolute referential.

Reviewer: ```suggestion
 |{var}float waterline_to_chart_datum(ping_time) |O |Vertical translation vector at the time of the ping matching the distance from the water line to the chart data reference (typically Lowest Astronomical Tide or Mean Sea Level). This variable is the vector that contains the tide and allows for the positioning of samples in an absolute reference system.
```
